code128/encode.go

```diff
@@ -180,7 +180,12 @@ func Encode(content string) (barcode.BarcodeIntCS, error) {
 	sum = sum % 103
 	result.AddBit(encodingTable[sum]...)
 	result.AddBit(encodingTable[stopSymbol]...)
-	return utils.New1DCodeIntCheckSum(barcode.TypeCode128, content, result, sum), nil
+	return utils.New1DCodeIntCheckSumWithColor(barcode.TypeCode128, content, result, sum, color), nil
+}
+
+// Encode creates a Code 128 barcode for the given content
+func Encode(content string) (barcode.BarcodeIntCS, error) {
+	return EncodeWithColor(content, barcode.ColorScheme16)
 }
 
 func EncodeWithoutChecksum(content string) (barcode.Barcode, error) {
```

Comment:
This Encoder is missing a color overload

Reply:
I didn't understand what you meant. Apparently this encoder is the same as the others. Encode calls EncodeWithColor passing barcode. ColorScheme16 in a fixed way. I did not change the expected parameters of the Encode method to maintain compatibility.

Reply:
I was talking about the `EncodeWithoutChecksum` overload. Imho there should also be a variant to encode with color and without checksum

Reply:
Overload created in [commit a8e67c5](https://github.com/boombuler/barcode/pull/84/commits/a8e67c5d16f5f4de67a62c2dd28d48f9e52216c6)